OpenRA.Mods.Common/Commands/PlayerCommands.cs

```diff
@@ -35,19 +35,32 @@ public void WorldLoaded(World w, WorldRenderer wr)
 
 		public void InvokeCommand(string name, string arg)
 		{
+			if (world.LocalPlayer == null)
+				return;
+
+			if (world.LocalPlayer.WinState != WinState.Undefined)
+			{
+				Game.Debug("This command ({0}) is disabled.".F(name));
```

Comment:
It would be better to silently return here for consistency with the other ways we allow pausing and surrendering.